### PR TITLE
num_arrays_in_memory

### PR DESCRIPTION
**Pull request by tamukagate**

### CODE REVIEW

**Comment by tamukagate** [comment]
add_reformatting_checklist_general

**Comment by tamukagate** [comment]
Close #7113

File: ivy/functional/ivy/general.py [hunk 1]
```diff
@@ -2050,7 +2050,21 @@ def get_all_arrays_in_memory():
 
 @handle_exceptions
 def num_arrays_in_memory():
-    """Returns the number of arrays which are currently alive."""
+    """Returns the number of arrays which are currently alive.
```
Comment (rahulsnkr) [inline ivy/functional/ivy/general.py:2053]: @tamukagate thanks for your contribution! I can see that the docstring tests are failing for the changes you made. I'll merge the PR once they are fixed :)

**Review by rahulsnkr — CHANGES_REQUESTED** [review]

File: ivy/functional/ivy/general.py (hunk 1 above)
Comment (tamukagate) [reply ivy/functional/ivy/general.py:2053]: Hi @rahulsnkr. I'm currently stuck on the implementation of the docstring examples. Please may you help me

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

File: ivy/functional/ivy/general.py (hunk 1 above)
Comment (tamukagate) [reply ivy/functional/ivy/general.py:2053]: Hi @rahulsnkr, I've reverted the commits and rerun the tests

**Review by rahulsnkr — CHANGES_REQUESTED** [review]

**Pull request closed without merging**